infra/ansible/aggregator.yaml

```diff
@@ -0,0 +1,61 @@
+- import_playbook: setup.yaml
+- import_playbook: go.yaml
+- import_playbook: rust.yaml
+- import_playbook: eigenlayer-cli.yaml
+
+- hosts: aggregator
+  vars:
+    service: "aggregator"
+
+  tasks:
+    - name: Update apt and install required system packages
+      become: true
+      apt:
+        pkg:
+          - pkg-config
+          - libssl-dev
+          - gcc
+        state: latest
+        update_cache: true
+
+    - name: Create directories for each service if do not exist
+      become: true
+      ansible.builtin.file:
+        path: /home/{{ app_user }}/repos/{{ service }}
+        state: directory
+        mode: '0755'
+        owner: '{{ app_user }}'
+        group: '{{ app_user }}'
+      loop:
+        - aggregator
+
+    - name: Clone Aligned repository
+      become: true
+      ansible.builtin.git:
+        repo: https://github.com/yetanotherco/aligned_layer.git
+        dest: /home/{{ app_user }}/repos/{{ service }}/aligned_layer
+        version: v0.10.2
+      loop:
+        - aggregator
+
+    - name: Set permissions for cloned repository
+      become: true
+      ansible.builtin.file:
+        path: /home/{{ app_user }}/repos/{{ service }}/aligned_layer
+        mode: '0755'
+        owner: '{{ app_user }}'
+        group: '{{ app_user }}'
+        recurse: yes
+
+    - name: Compile all FFIs
+      become: true
+      shell:
+        chdir: /home/{{ app_user }}/repos/aggregator/aligned_layer
+        cmd: sudo -u {{ app_user }} . /home/{{ app_user }}/.profile && sudo -u {{ app_user }} make build_all_ffi_linux
```

Comment:
Instead of using `sudo -u`, why not use `become: {{ app_user }}`?

Reply:
This file will be refactored in  #1387